app/Http/Controllers/ReportController.php

```diff
@@ -8,23 +8,23 @@
 
 class ReportController extends Controller
 {
-    public function index(){
+    public function index() {
         $r = UserReport::all();
 
         return view('reports.index', [
             'reports' => $r,
         ]);
     }
 
-    public function create_game_report($gameid){
+    public function create_game_report($gameid) {
```

Comment:
__[code_sniffer]__
* Method name "ReportController::create_game_report" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->